frontend/src/features/consultation/ui/chat-panel.module.css

```
@@ -105,42 +142,44 @@
 .msgBubble {
-  padding: 12px 16px;
-  border-radius: 18px;
-  font-size: 0.9rem;
-  letter-spacing: -0.1px;
-  line-height: 1.5;
+  padding: 10px 14px;
+  border-radius: 14px;
+  font-size: 0.85rem;
+  line-height: 1.45;
+  max-width: 75%;
   word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)